nodle-parachain/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "subql-dictionary",
+  "version": "1.0.0",
+  "description": "A SubQuery Dictionary Project that provides increased indexing speed to all projects",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc -b",
```

We're shifting to new build tooling, can you please try this
```suggestion
    "build": "subql build",
```